docs/developer/.scripts/33_render_status.py

```diff
@@ -137,9 +137,21 @@ def render_metadata_progress():
 
 def render_logs_progress():
     not_possible = {
-        'sap_hana'  # https://github.com/DataDog/architecture/blob/master/rfcs/agent-integrations/sap_hana.md#open-questions
+        'sap_hana',  # https://github.com/DataDog/architecture/blob/master/rfcs/agent-integrations/sap_hana.md#open-questions
+        'ntp',  # the integration is for a remote ntp server
+        'btrfs',  # it emits to the system log
+        'http_check',  # Its not a service
+        'snmp',  # remote connection to the devices
+        'openmetrics',  # base class
+        'pdh_check',   # base class
+        'prometheus',  # base class
+        'snowflake',  # No logs to parse, needs to be from QUERY_HISTORY view
+        'vsphere',
```

Comment:
vSphere is possible. Tricky and very long as it's unclear how many log formats there are + it requires to setup a syslog server